hadoop-tools/hadoop-azure/.gitignore

```diff
@@ -1,2 +1,4 @@
 .checkstyle
-bin/
\ No newline at end of file
+bin/
+testlogs
```

Comment:
test results should go into target so a "mvn clean" will delete them

Reply:
Done